apps/app/src/components/Sidebar/SideBar2.tsx

```diff
@@ -0,0 +1,101 @@
+import { useRouter } from 'next/router'
+import React from 'react'
+import { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useAppPersistStore, useAppStore } from 'src/store/app'
+import { useDisconnect } from 'wagmi'
+
+import getAvatar from '@/lib/getAvatar'
+
+import TabTitle from './TabTile'
+
+export interface ITabProps {
+  icon: React.ReactElement
+  label: string
+}
+
+interface ISidebarProps {
+  emptyTop?: boolean
+  selectedIndex: number
+  setSelectedIndex: (i: number) => void
+  tabs: ITabProps[]
+}
```

Comment:
could you import these from the other sidebar component instead of re-declaring them